meta/saiserialize.c

```diff
@@ -576,10 +576,38 @@ int sai_deserialize_macsec_auth_key(
         return read;
     }
 
-    SAI_META_LOG_WARN("failed to deserialize '%.*s' as macsec_auth_key", MAX_CHARS_PRINT, buffer);
+    SAI_META_LOG_WARN("failed to deserialize '%.*s' as auth_key", MAX_CHARS_PRINT, buffer);
     return SAI_SERIALIZE_ERROR;
 }
 
+int sai_serialize_macsec_sak(
+        _Out_ char *buffer,
+        _In_ const sai_macsec_sak_t sak)
+{
+   return sai_serialize_encrypt_key( buffer, sak );
```

Comment:
please remove space before buffer and after sak

Reply:
Done